README.md

```diff
@@ -19,9 +19,9 @@ More arguments are available - check them out with `itch-dl --help`.
 
 The downloader is able to grab more or less everything you can download via the itch app.
 
-The input URL can be any "Browse" page (top, popular, newest, filtered by tags, etc) or any
-game jam. The input can also be a path to a itch.io JSON file with game jam entries, or just
-a list of itch.io game URLs (not browse/jam pages!) to download.
+The input URL can be any "Browse" page (top, popular, newest, filtered by tags, etc), any
+game jam or any bundle. The input can also be a path to a itch.io JSON file with game jam
```

Comment:
I'd like some clarification that this does not claim bundles - we'll handle this later.